src/router/routes.js

```diff
@@ -12,7 +12,8 @@ module.exports = [
 
 				return exp?.version === 'b' ? 'hp/crowdfund-for-good' : 'home';
 			},
-			unbounce: true,
+			unbounceEmailCapture: true,
+			unbounceDonation: true,
```

Comment:
Is there something that will cancel out one of these? Otherwise someone with no unbounce cookie hitting the homepage would get 2 popups...

Reply:
This will be the responsibility of product with the Unbounce rules.